IDEAS/Buildings/Components/InternalWall.mo

```diff
@@ -84,7 +72,15 @@ model InternalWall "interior opaque wall between two zones"
 
   parameter Real CD=0.78 "Discharge coefficient of cavity"
     annotation (Dialog(tab="Airflow"));
+  final parameter Real hzone_b(fixed=false);
+  final parameter Real hfloor_b(fixed=false);
+
+  parameter Real hRef_b=if inc== Modelica.Constants.pi then hzone_b else 0  "Height above the zone floor at propsbus_b. Height where the surface starts. e.g. 0 for walls at floor level and floors.  ";
```

Comment:
Can you use `IDEAS.Utilities.Math.Functions.isAngle` for all angle comparisons? It's a bit more forgiving when roundoff errors occur and it considers that angle 0 equals angle 360.